scripts/mkproject.bat

```diff
@@ -18,7 +18,7 @@ goto END
 
 :MKPROJECT
 if not defined PROJECT_HOME (
-    set "PROJECT_HOME=%USERPROFILE%\.projects"
+    set "PROJECT_HOME=%USERPROFILE%\Projects"
```

Comment:
Do we need a default PROJECT_HOME? The *nix version says:
```
(dev)go|/srv$ mkproject
ERROR: Set the PROJECT_HOME shell variable to the name of the directory where projects should be created.
```

Reply:
I guess I don't really mind.  If a strict port is preferred then I can remove this functionality and replace this line with that error message instead.